module/surveys/helper.js

```diff
@@ -787,7 +788,8 @@ module.exports = class SurveysHelper {
                     userId,
                     validateSurvey.data.submissionId,
                     roleInformation,
-                    token
+                    token,
+                    type
```

Comment:
@swetha-aggidevara we have added this type everywhere not sure what is the use case if you can explain, please.